### PR TITLE
Bump version of collection and highlights list to support digests

### PR DESCRIPTION
**Pull request by nlisgo**
Improves: elifesciences/issues#4294

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Looks like a PHP-CS-Fixer update has started to find some new problems on existing code. We should have a separate PR to fix that first.

**Comment by nlisgo** [comment]
> Looks like a PHP-CS-Fixer update has started to find some new problems on existing code. We should have a separate PR to fix that first.

https://github.com/elifesciences/api-sdk-php/pull/191

**Comment by nlisgo** [comment]
There are a few instances of duplication of api versions. I have discussed this with @giorgiosironi and we agree that it would be great to take this on as part of the issue: elifesciences/issues#4294.

I have added a todo item.

File: test/Serializer/CollectionNormalizerTest.php [hunk 1]
```diff
@@ -163,6 +164,8 @@ public function normalizeProvider() : array
                     ->withRelatedContent(new ArraySequence([
                         Builder::for(ArticlePoa::class)
                             ->sample('growth-factor'),
+                        Builder::for(Digest::class)
+                            ->sample('neighbourhood-watch'),
```
Comment (thewilkybarkid) [inline test/Serializer/CollectionNormalizerTest.php:169]: Let's add an event here too.

**Comment by thewilkybarkid** [comment]
Looks good. We'll have to wait for the new versions to be available before merging.

File: test/Serializer/CollectionNormalizerTest.php (hunk 1 above)
Comment (nlisgo) [reply test/Serializer/CollectionNormalizerTest.php:169]: > Let's add an event here too.

Done

File: test/Builder.php [hunk 2]
```diff
@@ -140,6 +141,20 @@ private function defaultTestData()
...
                 },
+                Event::class => function () {
+                    return [
+                        'id' => '1',
+                        'title' => 'Event 1 title',
+                        'impactStatement' => null,
+                        'publishedDate' => null,
+                        'updatedDate' => null,
+                        'starts' => null,
+                        'ends' => null,
```
Comment (thewilkybarkid) [inline test/Builder.php]: This shouldn't contain invalid data.

**Pull request merged**